zipline-gradle-plugin/build.gradle.kts

```diff
@@ -21,11 +25,27 @@ buildConfig {
 
 gradlePlugin {
   plugins {
-    create("ktBridge") {
-      id = Ext.kotlinPluginId as String
-      displayName = "KtBridge"
-      description = "Generate bridges for calling from Kotlin/JVM to Kotlin/JS"
-      implementationClass = "app.cash.zipline.ktbridge.plugin.KtBridgeGradlePlugin"
+    create("zipline") {
+      id = "app.cash.zipline"
+      displayName = "zipline"
+      description = "Compiler plugin to generate bridges between platforms"
+      implementationClass = "app.cash.zipline.gradle.ZiplinePlugin"
     }
   }
 }
+
+// The plugin marker artifacts don't conform to Maven Central's requirements. Disable publishing
```

Comment:
This isn't a problem in licensee. Why is it here?

Reply:
I get it this in Wire:

> Invalid POM: /com/squareup/wire/com.squareup.wire.gradle.plugin/4.0.0-alpha.12/com.squareup.wire.gradle.plugin-4.0.0-alpha.12.pom: Project name missing, Project description missing

Most likely cause is the publish plugin doesn’t create a complete POM for these.

Reply:
Found it!
https://github.com/square/wire/pull/2066/files